### PR TITLE
feature_AUSMplusup

### PR DESCRIPTION
**Pull request by aeroamit**
## Proposed Changes
AUSM+-up scheme of Liou has been implemented (A sequel to AUSM, Part II: AUSM + -up for all speeds).
- This scheme is more robust and accurate in comparison to existing AUSM scheme in the code
- Relatively fewer issues in comparison to Roe scheme
- Proper scaling of numerical dissipation at lower Mach numbers
- I tried it over a relatively complex geometry, implementation seems to be fine.

 

## PR Checklist
*Put an X by all that apply. You can fill this out after submitting the PR. If you have any questions, don't hesitate to ask! We want to help. These are a guide for you to know what the reviewers will be looking for in your contribution.*

- [x] I am submitting my contribution to the develop branch.
- [x] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags).
- [x] My contribution is commented and consistent with SU2 style.
- [] I have added a test case that demonstrates my contribution, if necessary.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,11 +12,11 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - aero_amit@hotmail.com
```
Comment (EduardoMolina) [inline .travis.yml]: Can you please revert this back to the original?

File: .travis.yml [hunk 2]
```diff
@@ -12,11 +12,11 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - aero_amit@hotmail.com         
   
 branches:
     only:
-        - develop
+        - AUSMPLUSUP
```
Comment (EduardoMolina) [inline .travis.yml]: This also.

Comment (aeroamit) [reply .travis.yml]: Reverted back this as well.

Thanks@EduardoMolina

**Comment by EduardoMolina** [comment]
Thanks @aeroamit 
It is time to merge.

**Pull request merged**